assignments/life_expectancy/tests/__init__.py

```diff
@@ -2,5 +2,7 @@
 from pathlib import Path
 
 
-FIXTURES_DIR = Path(__file__).parent / "fixtures"
-OUTPUT_DIR = Path(__file__).parent.parent / "data"
+PROJECT_DIR = Path(__file__).parents[2]
```

Comment:
why `[2]` ?

Reply:
Since we are in `assignments/life_expectancy/tests/` the alternative would be `Path(__file__).parent.parent.parent` and I thought that would be too many parents! So Path.parent**s**[tree node index] is a more concise alternative